client/blocks/product-purchase-features/product-purchase-features-list/index.jsx

```diff
@@ -147,7 +148,7 @@ export default class ProductPurchaseFeaturesList extends Component {
 			sitePlans,
 		} = this.props;
 
-		const plan = find( sitePlans.data, isWpcomPremium );
+		const plan = find( sitePlans.data, isWpcomPersonal );
```

Comment:
@lamosty this is out of scope, but do you remember why this component isn't connected anymore? We have a number of selectors we could use otherwise, like `getSitePlan` and `hasDomainCredit`